app/src/main/kotlin/com/saltedge/authenticator/features/settings/list/SettingsListFragment.kt

```diff
@@ -102,7 +102,15 @@ class SettingsListFragment : BaseFragment(), SettingsListContract.View,
     }
 
     override fun showInfo(message: Int) {
```

Comment:
now this function has single responsibility, to ask user to restart the app. Please rename it and remove `message` argument.